src/Hosting/Server.Abstractions/src/Features/IServerAddressesFeature.cs

```diff
@@ -5,10 +5,19 @@
 
 namespace Microsoft.AspNetCore.Hosting.Server.Features
 {
+    /// <summary>
+    /// Specifies the address used by the server.
+    /// </summary>
     public interface IServerAddressesFeature
     {
+        /// <summary>
+        /// A <see cref="ICollection{T}" /> of addresses used by the server.
+        /// </summary>
         ICollection<string> Addresses { get; }
 
+        /// <summary>
+        /// <c>true</c> to prefer URLs configured on the <see typeparamref="Microsoft.AspNetCore.Hosting.IWebHostBuilder"/>; otherwise <c>false</c>.
```

Comment:
```suggestion
        /// <see langword="true" /> to prefer URLs configured on the <see cref="Microsoft.AspNetCore.Hosting.IWebHostBuilder"/>; otherwise <see langword="false" />.
```

Reply:
```suggestion
        /// <see langword="true" /> to prefer URLs configured by the host rather than the server.
```